src/game/client/hud_crosshair.cpp

```diff
@@ -390,7 +414,7 @@ void CHudCrosshair::Paint( void )
 
 	// NEO TODO (nullsystem): Probably be better if the entire class refreshed to our own
 	// thing and can do away with that CHudTexture nonsense entirely
-	const bool bIsScoped = pNeoWep && pNeoWep->GetNeoWepBits() & NEO_WEP_SCOPEDWEAPON;
+	const bool bIsScoped = pWeapon && pWeapon->GetNeoWepBits() & NEO_WEP_SCOPEDWEAPON;
```

Comment:
Nit: Bit of a misnomer; could we rename this `bIsScopedWep` or something, because it's not about whether or not the player "is scoped", but rather the abilities of their equipped gun (whether or not the gun is able to scope, regardless of current scope status).